RoadPricingAngular/src/app/user.service.ts

```diff
@@ -43,4 +45,26 @@ export class OwnerService {
     return this.httpService.get(`${API_URL_LOCALHOST}/owner/${ownerId}/cars`);
   }
 
+  /**
+   * Register an Owner.
+   * @param owner The newly created owner
+   */
+  register(owner: Owner) {
+    //Send request to the administration backend. /owner/create with owner in the body.
```

Comment:
Comment klopt niet meer / is niet recent net zoals de JSdoc hierboven, account / owner wordt door elkaar gebruikt.